.rubocop_todo.yml

```diff
@@ -38,53 +49,63 @@ Lint/UnusedBlockArgument:
 Lint/UnusedMethodArgument:
   Enabled: false
 
-# Offense count: 508
+# Offense count: 3
+# Configuration parameters: ContextCreatingMethods.
+Lint/UselessAccessModifier:
+  Enabled: false
```

Comment:
checked those 3, looks like there is a bug in rubocop that mistakenly marks them as useless. I'll leave it.